alphabase/psm_reader/msfragger_reader.py

```diff
@@ -48,6 +52,12 @@ def _get_mods_from_masses(sequence, msf_aa_mods):
                         site_str = '0'
                     else:
                         _mod = mod_name.split('@')[0]+'@'+sequence[0]
+                elif site==cterm_position:
+                    if mod_name.endswith('C-term'):
+                        _mod = mod_name
+                    else:
+                        _mod = mod_name.split('@')[0]+'@Any C-term' #what if only Protein C-term is listed?
```

Comment:
We can check whether `modname@Any C-term` or `modname@Protein C-term` is in the `MOD_MASS` dict